### DISCUSSION TITLE
Building without WIX - does it make sense?

### DISCUSSION THREAD

**Discussion by abitrolly** [RHVoice/RHVoice#229] [Q&A] [answered]
What is needed to run `RHVoice` without installing? I tried to build it without NSIS and WIX and it fails while trying to build the data. Would be nice to get a `README.md` in https://github.com/Olga-Yakovleva/RHVoice/tree/master/data describing what are these files, what is the result of building them, and where `RHVoice` expects to find them (if it doesn't find them by default).

Right now the build without `WIX` fails.
```python
...
Checking for pkg-config... no
KeyError: 'WindowsInstallers_core_64':
  File "C:\projects\rhvoice\SConstruct", line 357:
    build_for_windows(base_env,vars)
  File "C:\projects\rhvoice\SConstruct", line 340:
    duplicate=0)
  File "c:\python27\lib\site-packages\scons\SCons\Script\SConscript.py", line 660:
    return method(*args, **kw)
  File "c:\python27\lib\site-packages\scons\SCons\Script\SConscript.py", line 597:
    return _SConscript(self.fs, *files, **subst_kw)
  File "c:\python27\lib\site-packages\scons\SCons\Script\SConscript.py", line 286:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "C:\projects\rhvoice\data\SConscript", line 79:
    packagers["exe"].msis.append(local_env["WindowsInstallers_core_64"])
  File "c:\python27\lib\site-packages\scons\SCons\Environment.py", line 405:
    return self._dict[key]
Command exited with code 2
```

https://ci.appveyor.com/project/yakshaveinc/rhvoice/builds/36206042

**Comment by zstanecic** [comment]
Hi,

It builds the MSI installers, which are needed for the successful installation of the engine and voices.

Best,

Zvonimir

**Comment by abitrolly** [comment]
Is it possible to build and use RHVoice on Windows in portable way without installers?

**Comment by zstanecic** [comment]
No, it is not possible, because it works as the sapi component.

**Comment by alex19EP** [comment, marked as answer]
> what are these files?

this is a just data files used by RHVoice.

> what is the result of building them?

they just included in installer. the build system doesn't change them in any way.
